style.css

```
@@ -0,0 +1,55 @@
+* {
```

This `*` selector is applying your background color to every single element on the page. While that gets the look you want, it's also a bit overdone. You can instead replace this with the `body` selector to get the same effect:
```suggestion
body {
```